go/vt/sqlparser/ast_rewriting.go

```diff
@@ -110,7 +112,7 @@ const (
 )
 
 func (er *expressionRewriter) rewriteAliasedExpr(node *AliasedExpr) (*BindVarNeeds, error) {
-	inner := newExpressionRewriter()
+	inner := newExpressionRewriter("")
```

Comment:
should this string be `er.keyspace` instead of ""?

Reply:
I see no harm. Fixed.